requirements.txt

```diff
@@ -7,4 +7,8 @@ Authlib
 pylint
 scikit-learn
 numpy
-google-api-python-client
\ No newline at end of file
+tensorflow
+tensorflow-gpu
```

Comment:
When I try to run  `pip3 install -r requirements.txt` the tensorflow-gpu module can not be found. Returning these errors: `ERROR: Could not find a version that satisfies the requirement tensorflow-gpu (from versions: none)` and
`ERROR: No matching distribution found for tensorflow-gpu`